### DISCUSSION TITLE
Raising the limit of active playlists

### DISCUSSION THREAD

**Discussion by DarkbitNetwork** [ppy/osu#24587] [Ideas] [answered]
_Copied from official osu! discord server_
Hello, I was trying to make two playlists active (first for easier ver, second for harder ver) at the same time but discovered that it's not possible to do so.

My question is if there is any plan (or possibility) of raising the amount of playlists you can have at the same time?

**Comment by bdach** [comment, marked as answer]
Higher limit of active rooms at a time is currently a supporter feature. Without supporter you may only have one playlist active; with supporter this limit goes up to 5 active playlists at a time.

**Reply by DarkbitNetwork** [reply]
Thank you for your answer, I wasn't aware of supporter increasing the limit of active playlists because there's nothing telling me that I need supporter to increase the limit.
